### DISCUSSION TITLE
clone from git

### DISCUSSION THREAD

**Discussion by Rshiya** [google/model-viewer#2103] [General]
i tried to run npm bootstrap command and i get the following error
./node_modules/.bin/lerna bootstrap --ci
' . ' is not recognized as an internal or external command
i tried to add model-viewer folder path in environment variables  like this C:\Users\MYPC\model-viewer
but still getting same error
can u help me please??

**Comment by elalish** [comment]
Sorry, we develop on MacOS and Linux, so the commands have not been been tested on Windows. Others have gotten it working under WSL, though the setup is not entirely trivial as I recall. Apologies.
